.env.example

```diff
@@ -27,4 +27,4 @@ SLACK_API_TOKEN_WORKSPACE_BOT=xoxb-54321-secret
 
 GATSBY_TEST_STRIPE_PUBLISHABLE_KEY=pk_test_secret
 
-DATABASE_URL=postgresql://USER:PASSWORD@HOST:PORT/DATABASE
+DATABASE_URL='postgres://postgres:JmHY3J%2B4%5Ej8hGq5r%2CQ%3F_aJhBOm4Gj%21~y14@db.lzrlswbnxaguztrhabui.supabase.co:6543/postgres?pgbouncer=true'
```

Comment:
whoops, this should not have been put in the example, i probably should reset the password now, the russians might have it by now.